ansible/roles/operator-pipeline/tasks/main.yml

```diff
@@ -183,4 +183,6 @@
                     command: ["tkn", "pipelinerun", "delete", "--keep", "{{ preserve_after_cleanup }}"]
                 restartPolicy: OnFailure
 
-- include: tasks/operator-hosted-pipeline-trigger.yml
+- include: tasks/webhook-event-listener.yml
+- include: tasks/webhook-hosted-pipeline-trigger.yml
+- include: tasks/webhook-release-pipeline-trigger.yml
```

Comment:
We can do this as a followup but should we move all these into a more appropriately named directory since they're not tasks? Perhaps `triggers`.

Reply:
The name `tasks` exists here in the context of Ansible, as an "ansible thing to do"
https://docs.ansible.com/ansible/latest/user_guide/playbooks_intro.html#task-execution
We have this convention in other projects too
https://gitlab.cee.redhat.com/isv/mercury/mercury/-/tree/master/ansible/roles/mercury/tasks